protocols/kad/src/behaviour.rs

```diff
@@ -946,8 +946,6 @@ where
             return
         }
 
-        let now = Instant::now();
```

Comment:
I think it is preferable to leave this here as it can be reused multiple times (cf. https://github.com/libp2p/rust-libp2p/pull/1496). And since `Instant::now()` is a side-effecting function for obtaining some notion of system time, usually both testability and performance is improved by using it mostly in top-level functions and passing the instant down to others where needed (see also the other comments).